go/chisel/server/server_handler.go

```diff
@@ -253,22 +278,165 @@ func (s *Server) handleDynReverse(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+var baseFingerbankPort = 23000
+var maxCheckedInConnectors = 256
+
 func (s *Server) handleRemoteBinds(w http.ResponseWriter, req *http.Request) {
-	managementNetwork := pfconfigdriver.Config.Interfaces.ManagementNetwork
-	pfconfigdriver.FetchDecodeSocket(req.Context(), &managementNetwork)
+	connectorId := req.URL.Query().Get("connector-id")
+	if connectorId == "" {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(unifiedapiclient.ErrorReply{Status: http.StatusNotFound, Message: "Missing connector-id query parameter"})
+		return
+	}
+
+	if o, ok := activeTunnels.Load(connectorId); ok {
+		tun := o.(*tunnel.Tunnel)
+		index := s.computeConnectorIndex(connectorId)
```

Comment:
It looks like this index only increases.

Reply:
It does, I'm assuming that we're connecting less than 256 connectors to a single server which seems like a fair limit for the moment

Reply:
Can connectors be removed?

Reply:
They can but it's pretty unlikely a lot of them get removed. Worst case, the service can be restarted to clear the counter